lib/jekyll-github-metadata/repository.rb

```diff
@@ -98,6 +98,10 @@ def releases
         memoize_value :@releases, Value.new(proc { |c| c.releases(nwo) })
       end
 
+      def commits
+        memoize_value :@commits, Value.new(proc { |c| c.commits(nwo) })
```

Comment:
Yeaaa... 200,000 commits are definitely not needed. Are you able to set `auto_paginate` to `false` for a specific endpoint?